pkg/cli/scan.go

```diff
@@ -127,14 +128,55 @@ func Scan() *cobra.Command {
 }
 
 func scanInput(inputFilePath string, p *scanParams) (*inputScan, error) {
+	if inputFilePath == "-" {
+		// Read stdin into a temp file.
+		t, err := os.CreateTemp("", "wolfictl-scan-")
+		if err != nil {
+			return nil, fmt.Errorf("failed to create temp file for stdin: %w", err)
+		}
+		if _, err := io.Copy(t, os.Stdin); err != nil {
+			return nil, err
+		}
+		if err := t.Close(); err != nil {
+			return nil, err
+		}
+		fmt.Fprintln(os.Stderr, "Will process apk from stdin")
+		inputFilePath = t.Name()
+	} else if strings.HasPrefix(inputFilePath, "https://") {
+		// Fetch the remote apk into a temp file.
+		t, err := os.CreateTemp("", "wolfictl-scan-")
+		if err != nil {
+			return nil, fmt.Errorf("failed to create temp file for remote apk: %w", err)
+		}
+		resp, err := http.Get(inputFilePath) //nolint:gosec
```

Comment:
Just curious what the `//nolint:gosec` is dealing with

Reply:
golangci-lint complained that a user-provided value was passed to `http.Get`, and I just don't care. 🤷

Reply:
Gotcha! Cool, yeah we intend for this to be an arbitrary remote thing so that makes sense!